cmd/links-add-capella.go

```diff
@@ -0,0 +1,51 @@
+package cmd
+
+import (
+	"github.com/couchbaselabs/cbdinocluster/deployment/clouddeploy"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+)
+
+var linksCapellaCmd = &cobra.Command{
```

Comment:
I'm slightly suspicious of this being specific to Columnar since I believe these are really just Analytics endpoints.  It looks like the implementation uses the Capella control-plane proxies to set it up though.  What is the use-case for needing to set these up as part of dinocluster rather than a specific test?

Reply:
For the capella link I would've thought its not possible using the analytics enpoint, as all you have to provide in capella columnar is a capella cluster id, and it links it. If you were to link it as if it were a separate couchbase cluster then you would have to provide all the conneciton details I imagine, which could work.

The S3 one is possible just using the analytics endpoint so yeah might be better to just use that in the tests directly.

Reply:
Based on there being some Columnar specific details here, we can approve this and consider any improvements to make this more generic as future-improvements.

Reply:
Sounds good, thanks.